web/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "parser": "babel-eslint",
-  "extends": ["airbnb", "prettier", "plugin:jsx-a11y/strict", "plugin:cypress/recommended"],
+  "extends": ["airbnb", "prettier", "plugin:jsx-a11y/strict"],
```

Comment:
could we create a .eslintrc.json file for integrationTests with that plugin in it?
```
{
  "parser": "babel-eslint",
  "extends": ["airbnb", "prettier", "plugin:cypress/recommended"],
  "rules": {
    "arrow-body-style": "off",
    "no-param-reassign": [2, { "props": false }],
    "react/forbid-prop-types": ["error", { "forbid": ["any"] }],
    "react/jsx-filename-extension": 0,
    "react/jsx-fragments": ["error", "element"],
    "react/jsx-props-no-spreading": 0,
    "react/no-array-index-key": "warn"
  },
  "env": {
    "browser": true
  }
}
```

Reply:
I have the linter running. It shows quite a few issues.  I will fix those in a clean commit.